spinalcordtoolbox/deepseg/inference.py

```diff
@@ -126,7 +134,7 @@ def segment_non_ivadomed(path_model, model_type, input_filenames, threshold, rem
     return im_lst, target_lst
 
 
-def segment_monai(path_img, tmpdir, predictor):
+def segment_monai(path_img, tmpdir, predictor, device):
```

Comment:
Since there's no other indication of the expected type for `device`, would an annotation be good here?
```suggestion
def segment_monai(path_img, tmpdir, predictor, device: torch.device):
```
(I'm also not sure about the expected type of `predictor`, but that one looks more polymorphic so maybe we don't want to pin it down.)